File: SudokuSolver.Models/Analytics/Move.cs
namespace SudokuSolver.Models.Analytics
{
    public class Move
    {
        public int Order { get; set; }
        public int I { get; set; }
        public int J { get; set; }
        public int Value { get; set; }
        public SolverStrategyType Type { get; set; }
    }
}

File: sudokusolver.reactapp/src/models/Move.ts
import { SolverStrategyType } from "./SolverStrategyType";

export class Move{
    order: number = -1;
    i : number = -1;
    j: number = -1;
    value: number = -1;
    type: SolverStrategyType = SolverStrategyType.None;
}